rmarkdown.Rmd

```diff
@@ -118,9 +182,19 @@ Z
 >
 > > Block quote 
 
+#### Exercise 2: hands on!
+
+Time: 10 min
+
+Now you are asked to include in your "skecthed article" what we just have learned.
+
+1. Include a picture with caption (of your research, or any .png you find in your PC).
+2. Include a footnote.
+3. Include the link of au.code webpage to your document.
```

Comment:
"link to". Also, AU Open Coders. Or just AU Coders. Or `au-oc`, since that is the website prefix.